command_line/stills_process.py

```diff
@@ -1021,7 +1021,9 @@ def find_spots(self, experiments):
         logger.info("*" * 80)
 
         # Find the strong spots
-        observed = flex.reflection_table.from_observations(experiments, self.params)
+        observed = flex.reflection_table.from_observations(
```

Comment:
Codecov notes that this is not covered by the tests: I think this is a valid concern 🙂

I wonder if we could add a very small test for this code?

Reply:
Added!  The new test triggers #1180 on master but not on the `dsp_fix_grid_scan` branch.  I don't think Codecov will pick it up as it uses a external call to stills process.